src/lib/util/cdrom.cpp

```diff
@@ -455,7 +449,7 @@ chd_error read_partial_sector(cdrom_file *file, void *dest, uint32_t lbasector,
 	// if this is pregap info that isn't actually in the file, just return blank data
 	if (!phys)
 	{
-		if ((file->cdtoc.tracks[tracknum].pgdatasize == 0) && (lbasector < (file->cdtoc.tracks[tracknum].logframeofs + file->cdtoc.tracks[tracknum].pregap)))
+		if (file->cdtoc.tracks[tracknum].pgdatasize == 0 && lbasector < file->cdtoc.tracks[tracknum].logframeofs)
```

Comment:
The pregap is baked into logframeofs now so it shouldn't be added here. If it's added here then the audio will be silent during the beginning of the track. When the pregap section ends you can hear the sound suddenly cut in mid-track.